net/net-snmp/files/snmpd.init

```diff
@@ -284,6 +361,48 @@ snmpd_setup_fw_rules() {
 	HANDLED_SNMP_ZONES="$HANDLED_SNMP_ZONES $zone"
 }
 
+snmpd_configure_logging() {
+	local cfg="$1"
+	local log_syslog
+	local log_syslog_facility
+	local log_file
+	local log_file_priority
+
+	config_get_bool log_syslog "$cfg" log_syslog 0
+
+	# d - LOG_DAEMON,
+	# u - LOG_USER,
+	# 0-7 - LOG_LOCAL0 through LOG_LOCAL7.
+
+	# 0 or ! - LOG_EMERG
+	# 1 or a - LOG_ALERT
+	# 2 or c - LOG_CRIT
+	# 3 or e - LOG_ERR
+	# 4 or w - LOG_WARN
+	# 5 or n - LOG_NOTICE
+	# 6 or i - LOG_INFO
+	# 7 or d - LOG_DEBUG
+
+	if [ $log_syslog -eq 1 ]; then
+		config_get log_syslog_facility "$cfg" log_syslog_facility "d"
+
+		procd_append_param command -Ls "${log_syslog_facility}"
```

Comment:
oh, I see.  we're calling `procd_*` nested... personally I like having that all in the `service_start` function but I'm not going to insist.